### ISSUE TITLE
[Navigation] Null Pointer in NavigationImpl if navigation root missing jcr:content

### ISSUE THREAD

**Issue by ky940819** [adobe/aem-core-wcm-components#703]
## Bug Report

**Current Behavior**
If a navigation component has its navigation root property set to a page, and that page does not have a jcr:content child, then a NPE is thrown. This situation occurs sometimes when an author does not publish a parent page before publishing a child page. Although it is the result of user error, i do not believe that it should result in a 500 error.

```
Caused by: java.lang.NullPointerException: null
	at com.day.cq.wcm.core.impl.LanguageManagerImpl.getLanguageRootPath(LanguageManagerImpl.java:499) [com.day.cq.wcm.cq-wcm-core:5.11.276]
	at com.day.cq.wcm.core.impl.LanguageManagerImpl.getLanguageRootResource(LanguageManagerImpl.java:554) [com.day.cq.wcm.cq-wcm-core:5.11.276]
	at com.day.cq.wcm.core.impl.LanguageManagerImpl.getLanguageRoot(LanguageManagerImpl.java:537) [com.day.cq.wcm.cq-wcm-core:5.11.276]
	at com.day.cq.wcm.core.impl.LanguageManagerImpl.getLanguageRoot(LanguageManagerImpl.java:529) [com.day.cq.wcm.cq-wcm-core:5.11.276]
	at com.adobe.cq.wcm.core.components.internal.models.v1.NavigationImpl.getItems(NavigationImpl.java:106) [com.adobe.cq.core.wcm.components.core:2.5.0]
```

**Expected behavior/code**
Not a 500 error.

**Environment**
- AEM 6.4
- Core Components version 2.5.0
- JRE version 1.8.0

**Possible Solution**
Just wrapping the call in a null check would prevent the NPE.
However, the #getLanguageRootPath() method doesn't actually need the page's content resource, it should work fine with the resource for the actual page it's self (i.e. parent of the content resource). The resource located at the page path of the current page is very likely to exist.

**Comment by msagolj** [comment]
@ky940819 thanks for reporting this. As you have already looked into possible fix, could you maybe provide a PR for this ?

**Comment by msagolj** [comment]
@ky940819 thx for the fix :) has been merged, closing this issue.

**Comment by richardhand** [comment]
Reopening here as the issue still requires verification

**Comment by richardhand** [comment]
Code is merged to [master](https://github.com/adobe/aem-core-wcm-components/commits/master) and will be available in the upcoming [2.6.0 Release](https://github.com/adobe/aem-core-wcm-components/milestone/16).
